src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheckTest.java

```diff
@@ -167,16 +167,16 @@ public void testDefault3() throws Exception {
             "88:5: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 5),
             "94:5: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 5),
             "102:19: " + getCheckMessage(MSG_KEY_LINE_BREAK_AFTER, "{", 19),
-            "111:1: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 1),
-            "114:5: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 5),
-            "123:1: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 1),
-            "125:5: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 5),
-            "134:1: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 1),
-            "136:5: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 5),
-            "138:9: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 9),
-            "153:1: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 1),
-            "162:1: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 1),
-            "169:5: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 5),
+            "112:1: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 1),
```

Comment:
Please do all to avoid shifts in violations

Reply:
Done